OpenRA.Game/Map/Map.cs

```diff
@@ -1152,8 +1110,8 @@ public IEnumerable<CPos> FindTilesInAnnulus(CPos center, int minRange, int maxRa
 			if (maxRange < minRange)
 				throw new ArgumentOutOfRangeException("maxRange", "Maximum range is less than the minimum range.");
 
-			if (maxRange >= TilesByDistance.Length)
-				throw new ArgumentOutOfRangeException("maxRange", "The requested range ({0}) exceeds the maximum allowed ({1})".F(maxRange, MaxTilesInCircleRange));
+			if (maxRange >= Grid.TilesByDistance.Length)
+				throw new ArgumentOutOfRangeException("maxRange", "The requested range ({0}) exceeds the maximum allowed ({1})".F(maxRange, Grid.MaximumTileSearchRange));
```

Comment:
Actually, now that this is configurable, you might want to mention the setting in the message, something like "The requested range ({0}) cannot exceed the value of MaximumTileSearchRange ({1})."

(Similar thing in `Teleport.cs` below.